public/app/plugins/panel/pmm-update/components/CurrentVersion/CurrentVersion.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react';
+
+import { Icon, Tooltip, useStyles } from '@grafana/ui';
+
+import { useToggleOnAltClick } from '../../hooks';
+import { CurrentVersionProps } from '../../types';
+
+import { Messages } from './CurrentVersion.messages';
+import { getStyles } from './CurrentVersion.styles';
+
+export const CurrentVersion: FC<CurrentVersionProps> = ({ installedVersionDetails }) => {
+  const styles = useStyles(getStyles);
+  const [showFullVersion, handleToggleShowFullVersion] = useToggleOnAltClick(false);
+  const { installedVersionDate, installedVersion, installedFullVersion } = installedVersionDetails;
+
+  return (
+    <section className={styles.currentVersion}>
+      <span onClick={handleToggleShowFullVersion}>
+        {Messages.currentVersion}:{' '}
```

Comment:
Does the `{' '}` have the same function as the `nbsp` in the `AvailableUpdate` component?

Reply:
True. Changed to &nbsp